spec.yaml

```diff
@@ -109,6 +109,51 @@ paths:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
 
   /v4/clusters/:
+    get:
+      operationId: getClusters
+      tags:
+        - clusters
+      summary: Get clusters
+      description: |
+        This operation allows to fetch a list of clusters.
```

Comment:
You often take this sentence structure, but I think its grammatically incorrect.

It allows (SOMETHING) to do something.

`It allows to do something` isn't right I think.